sycl/include/CL/sycl/context.hpp

```diff
@@ -8,14 +8,9 @@
 
 #pragma once
 
-#include <CL/sycl/backend_types.hpp>
-#include <CL/sycl/detail/common.hpp>
-#include <CL/sycl/detail/export.hpp>
```

Comment:
Could you please clarify why inclusion of these headers is removed?

Reply:
I can restore them. They aren't used. But it's not a change that is necessary for this PR.

Reply:
In `context.hpp` we use `backend` from `backend_type.hpp`, from `common.hpp` we use `getSYCLImpl`-like function, from `export.hpp` we use `__SYCL_EXTERNAL`.
So, I think we need to restore them.

Reply:
done